Gems/AWSCore/Code/Include/Public/Framework/AWSApiJobConfig.h

```diff
@@ -136,7 +139,11 @@ namespace AWSCore
         Override<std::shared_ptr<Aws::Utils::RateLimits::RateLimiterInterface>> writeRateLimiter;
         Override<std::shared_ptr<Aws::Utils::RateLimits::RateLimiterInterface>> readRateLimiter;
         Override<Aws::Http::TransferLibType> httpLibOverride;
+#if AWSCORE_BACKWARD_INCOMPATIBLE_CHANGE
+        Override<Aws::Client::FollowRedirectsPolicy> followRedirects;
```

Comment:
To be clear, the expectation is that this is a temporary bandaid until all platforms are on the latest C++ SDK?

Reply:
correct